src/stirling/source_connectors/socket_tracer/protocols/pgsql/stitcher.cc

```diff
@@ -204,6 +207,10 @@ Status HandleQuery(const RegularMessage& msg, MsgDeqIter* resp_iter, const MsgDe
     }
   }
 
+  // TODO(ddelnano): This iterator incrementing should likely happen
+  // after the InvalidArgument error is returned below. This should only
+  // impact the case when there is an error, but we are leaving as is
+  // avoid a larger change without further verification.
```

Comment:
@etep I left this as a comment since I didn't get the unit test setup as quickly as I hoped.